components/tooltip/tooltip.js

```diff
@@ -470,7 +470,9 @@ class Tooltip extends RtlMixin(LitElement) {
 		window.addEventListener('resize', this._onTargetResize);
 
 		requestAnimationFrame(() => {
-			this._updateTarget();
+			if (this.isConnected) {
+				this._updateTarget();
+			}
```

Comment:
This came up due to the quickness that tooltips show/hide as elements receive focus. In this case, this rAF code was running after the tooltip had been removed from the DOM.